tasks/sep/base-001-MCP-L1/VALIDATION.md

```diff
@@ -0,0 +1,341 @@
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L38-L42). `0x78` is 120 in decimal, which matches the current value found by `cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 "SUBMISSION_INTERVAL()(uint256)" -r https://ethereum-sepolia.publicnode.com`.
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000005` <br/>
+  **Before:** `0x00000000000000000000000020044a0d104e9e788a0c984a2b7eae615afd046b` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000002` <br/>
+  **Meaning:** Sets `l2BlockTime` at slot `0x05` (5). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L45-L49). Units are in seconds, so the value should be 2 seconds to match the current value found by `cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 "L2_BLOCK_TIME()(uint256)" -r https://ethereum-sepolia.publicnode.com`.
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000006` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x000000000000000000000000da3037ff70ac92cd867c683bd807e5a484857405` <br/>
+  **Meaning:** Sets `challenger` at slot `0x06` (6). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L52-L56). This value matches the current address found by `cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 "CHALLENGER()(address)" -r https://ethereum-sepolia.publicnode.com`.
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000007` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x00000000000000000000000020044a0d104e9e788a0c984a2b7eae615afd046b` <br/>
+  **Meaning:** Sets `proposer` at slot `0x07` (7). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/L2OutputOracle.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/snapshots/storageLayout/L2OutputOracle.json#L59-L63). This value matches the current address found by `cast call 0x84457ca9d0163fbc4bbfe4dfbb20ba46e48df254 "PROPOSER()(address)" -r https://ethereum-sepolia.publicnode.com`.
+
+todo: this one differs from the template as well, the finalizationPeriodSeconds is set to 12 (0xc) on Base Sepolia, the template said: Units are in seconds, so the value should be `0x93a80` which is 604,800 in decimal.
```

Comment:
it looks like this is in fact the Base current value, so this difference seems ok